### PR TITLE
Croatian translation

### PR DESCRIPTION
**Pull request by milotype**

### CODE REVIEW

File: po/hr.po [hunk 1]
```diff
@@ -0,0 +1,400 @@
+# French translation for Money
```
Comment (nlogozzo) [inline po/hr.po:1]: French?

File: po/hr.po [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+# Copyright (C) 2022
+# This file is distributed under the same license as the org.nickvision.money package.
+# Enzo MAROS <enzo.maros@gmail.com>, 2022.
```
Comment (nlogozzo) [inline po/hr.po:4]: Please correct copyright

File: po/hr.po [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+#: src/ui/views/transactiondialog.cpp:121
+msgid "Amount (Empty)"
+msgstr "Iznos (prazan)"
+
+#: src/ui/views/transactiondialog.cpp:126
+msgid "Amount (Invalid)"
+msgstr "Iznos (neispravan)"
+
+#~ msgid "Currency"
+#~ msgstr "Monnaie"
```
Comment (nlogozzo) [inline po/hr.po:382]: Please finish these translations.

File: po/hr.po (hunk 2 above)
Comment (milotype) [reply po/hr.po]: Corrected!

**Comment by nlogozzo** [comment]
LGTM, thanks!

**Pull request merged**